src/selectors/trip_selector.js

```diff
@@ -0,0 +1,17 @@
+import createDeepEqualSelector from '@utils/cache_helper';
```

Comment:
let's call it `cacheHelper` instead. Also, try to use camelCase for the filename.